br/tests/br_restore_checkpoint/run.sh

```
@@ -103,19 +108,82 @@ export GO_FAILPOINTS=""
+rm -rf $TEST_DIR/$PREFIX/log/pitr_id_maps
+export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/restore/log_client/failed-after-id-maps-saved=return(true)"
+restore_fail=0
+run_br --pd $PD_ADDR restore point --full-backup-storage "local://$TEST_DIR/$PREFIX/full" -s "local://$TEST_DIR/$PREFIX/log" || restore_fail=1
+export GO_FAILPOINTS=""
+if [ $restore_fail -ne 1 ]; then
+    echo 'PITR success'
+    exit 1
+fi
+
+# check the pitr id map is saved in the log storage
+count=$(ls $TEST_DIR/$PREFIX/log/pitr_id_maps | wc -l)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)